.github/workflows/tox.yml

```diff
@@ -0,0 +1,27 @@
+name: "tox"
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+    - name: Test with tox
+      run: tox
+    - name: Upload Coverage to Codecov
```

Comment:
is codecov ready for this upload?

Reply:
I dropped this from the gh action, let's add that later if desired.